tests/core/emscripten_lazy_load_code.cpp

```diff
@@ -0,0 +1,43 @@
+#include <emscripten.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+extern "C" {
+
+EM_JS(void, log_started, (), {
+  out("foo_start");
+})
+
+EM_JS(void, log_ended, (), {
+  out("foo_end");
```

Comment:
Why not simply use puts/printf here?

Reply:
stdio increases code size, and it's nice in the test to only use it after the lazy loading, to see the size difference more clearly.